jekyll/_cci2_ja/server/v4.1/air-gapped-installation/phase-5-test-your-installation.adoc

```diff
@@ -3,37 +3,36 @@
 contentTags:
   platform:
   - Server v4.1
-  - Server Admin
+  - サーバー管理者
 ---
-= Phase 5 - Test your installation
+= フェーズ 5 - インストールのテスト
 :page-layout: classic-docs
 :page-liquid:
-:page-description: How to test your CircleCI server installation in an air-gapped environment.
+:page-description: エアギャップ環境にインストールした CircleCI Server をテストする方法について説明します。
 :icons: font
 :toc: macro
-
 :toc-title:
 
-Test your air-gapped installation using CircleCI's Reality Check project.
+CircleCI の Reality Check プロジェクトを使用して、エアギャップ環境にインストールした CircleCI Server をテストします。
 
 [#copy-reality-check]
-== 1.  Copy CircleCI server Reality Check
+== 1.  CircleCI Server Reality Check をコピーする
 
-Copy the link:https://github.com/circleci/realitycheck[CircleCI server Reality Check repository] to your GitHub Enterprise installation in your air-gapped environment.
+link:https://github.com/circleci/realitycheck[CircleCI Server Reality Check リポジトリ]を、エアギャップ環境の GitHub Enterprise インスタンスにコピーします。
 
 [#setup-reality-check]
-== 2. Set up CircleCI server Reality Check
+== 2.  CircleCI Server Reality Check を設定する
```

Comment:
There's in extra space between the numbers and text in some of the headings on this page